Description: fix bug in git commit 5e381e3
 In the new code a comma mysteriously turned into a period.
 This would have prevented proper separation
 of the mime type and content in data uris.
 Thanks to @hseg for catching this.
Origin: upstream, https://github.com/jgm/pandoc/commit/54561e9
Author: John MacFarlane <jgm@berkeley.edu>
Bug: https://github.com/jgm/pandoc/security/advisories/GHSA-xj5q-fv23-575g
Bug-Debian: https://security-tracker.debian.org/tracker/CVE-2023-35936
Forwarded: yes
Last-Update: 2023-07-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/Text/Pandoc/Class/IO.hs
+++ b/src/Text/Pandoc/Class/IO.hs
@@ -122,7 +122,7 @@
 openURL u
  | Just (URI{ uriScheme = "data:",
               uriPath = upath }) <- parseURI (T.unpack u) = do
-     let (mime, rest) = break (== '.') upath
+     let (mime, rest) = break (== ',') upath
      let contents = UTF8.fromString $ drop 1 rest
      return (decodeLenient contents, Just (T.pack mime))
  | otherwise = do
